### ISSUE TITLE
Replace the call to fetch users by email from identity to member api

### ISSUE THREAD

**Issue by vikasrohit** [topcoder-platform/tc-project-service#246]
As of now our new team management is not able to successfully send the invitation when user invites multiple emails. It is because we are using identity api to fetch existing users for the given email and identity actually does not support filtering by multiple emails. So, we need to replace it with member api's _search endpoint which supports the filtering by emails.

fyi @maxceem we need to get this resolved as soon as possible.

**Comment by maxceem** [comment]
@vikasrohit I think some time ago we've tried to replace identity service with member service for this purpose but couldn't. I cannot find where we had the discussion.

In short: member service does `loose` searching by email and returns many users who don't have the email we are searching for. 
For example, the next request for `3` emails finds `1 012 453` records while it returns `10 000` records per one request: https://api.topcoder.com/v3/members/_search/?fields=userId%2Chandle%2CphotoURL%2CfirstName%2ClastName%2Cdetails%2Cemail&query=email%3Amaxceem.dev@gmail.com%2Ctest@topcoder.com%2Ctest@yahoo.com&limit=10000

The `10 000` is the maximum `limit` size for such request, so we would have to make 102 requests to get all the records.

**Comment by maxceem** [comment]
I guess what we can do here is to make several requrests in parallel using identity service.

**Comment by vikasrohit** [comment]
Yep, I remember that @maxceem and I am in discussions with Member api team to have that feature in `_search` endpoint to allow us searching emails without tokenizing i.e. exact matches of the provided emails. However, that seems to be a long discussion and might not get implemented until we verify that there is no other way to achieve the requirements and we have identified final requirements around it.

> I guess what we can do here is to make several requrests in parallel using identity service.

This is one of solutions that surfaced during our discussions as well. We can definitely give a try as first fix for the problem because most of the time user would be inviting less than 10 emails only so that it would cause max of 10 parallel requests which should not be rate limited by our servers and at the same time should not flood the identity service as well.

So, in nutshell, @maxceem please create a separate issue for making parallel requests to the identity  when user invites more than one emails and get that implemented in our next release (if possible). We may introduce a configurable limit as well on number of emails to be invited at once.
